src/js/components/epics/detail.tsx

```diff
@@ -486,16 +554,62 @@ const EpicDetail = (props: RouteComponentProps) => {
                 canSubmit={project.has_push_permission}
                 handleAction={handleStepAction}
               />
+              <TourPopover
+                align="top"
+                heading={i18n.t('Wondering what to do next?')}
+                body={
+                  <Trans i18nKey="tourEpicNextSteps">
+                    The Next Steps section is designed as a quick reference to
+                    guide you through the process from adding your first Task to
+                    getting your Epic merged into the Project on GitHub. The
+                    next step is indicated with a blue ring, and completed steps
+                    are checked.
+                  </Trans>
+                }
+              />
             </div>
           </>
         }
       >
-        <EpicStatusPath status={epic.status} prIsOpen={epic.pr_is_open} />
+        <div className="slds-is-relative right-edge">
```

Comment:
@dvdherron are we using the right-edge class for anything?

Reply:
same as above. removed.